src/Std/Time/DateTime/PlainDateTime.lean

```diff
@@ -30,7 +30,13 @@ structure PlainDateTime where
   -/
   time : PlainTime
 
-  deriving Inhabited, BEq, Repr
+  deriving Repr
+
+instance : Inhabited PlainDateTime where
+  default := {
+    date := Inhabited.default,
+    time := Inhabited.default
```

Comment:
Why did you change this?

Reply:
I just removed some changes and forgot some parts of the code that could be smaller, In the next revisions I'm going to be more careful with it.